### PR TITLE
LPS-162049 LPS-165549 portal-search-web: Type - delete AssetEntriesSearchFacetTermDisplayContext

### PR DESCRIPTION
**Pull request by gustavolimav**
https://issues.liferay.com/browse/LPS-165549
https://issues.liferay.com/browse/LPS-165548

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by lipusz** [comment]
@gustavolimav Cab you make sure it also includes the deprecation of the old methods too as we have discussed today? Thanks!

/cc @joshuacords

**Comment by gustavolimav** [comment]
@lipusz deprecated in that [commit](https://github.com/liferay-search/liferay-portal/pull/691/commits/37ee1abeca92305e9aebde64eb142245dcb50234)

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/facets/view/asset_entries.jsp [hunk 1]
```diff
@@ -62,7 +62,7 @@ AssetEntriesSearchFacetDisplayContext assetEntriesSearchFacetDisplayContext = as
 				</li>
 
 				<%
-				for (AssetEntriesSearchFacetTermDisplayContext assetEntriesSearchFacetTermDisplayContext : assetEntriesSearchFacetDisplayContext.getTermDisplayContexts()) {
+				for (BucketDisplayContext assetEntriesSearchFacetTermDisplayContext : assetEntriesSearchFacetDisplayContext.getTermDisplayContexts()) {
```
Comment (joshuacords) [inline modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/facets/view/asset_entries.jsp]: @gustavolimav, you should also change assetEntriesSearchFacetTermDisplayContext to bucketDisplayContext

File: modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/type/facet/view.jsp [hunk 2]
```diff
@@ -85,7 +86,7 @@ TypeFacetPortletInstanceConfiguration typeFacetPortletInstanceConfiguration = as
 								<%
 								int i = 0;
 
-								for (AssetEntriesSearchFacetTermDisplayContext assetEntriesSearchFacetTermDisplayContext : assetEntriesSearchFacetDisplayContext.getTermDisplayContexts()) {
+								for (BucketDisplayContext assetEntriesSearchFacetTermDisplayContext : assetEntriesSearchFacetDisplayContext.getTermDisplayContexts()) {
```
Comment (joshuacords) [inline modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/type/facet/view.jsp]: Change here as well

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/facet/display/context/builder/AssetEntriesSearchFacetDisplayContextBuilder.java [hunk 3]
```diff
@@ -89,17 +89,17 @@ public AssetEntriesSearchFacetDisplayContext build() {
 		assetEntriesSearchFacetDisplayContext.setParameterValues(
 			_parameterValues);
 		assetEntriesSearchFacetDisplayContext.setRenderNothing(
-			ListUtil.isEmpty(termDisplayContexts));
-		assetEntriesSearchFacetDisplayContext.setTermDisplayContexts(
-			termDisplayContexts);
+			ListUtil.isEmpty(bucketDisplayContexts));
+		assetEntriesSearchFacetDisplayContext.setBucketDisplayContexts(
```
Comment (joshuacords) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/facet/display/context/builder/AssetEntriesSearchFacetDisplayContextBuilder.java:93]: This needs reordering

File: modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/facets/view/asset_entries.jsp (hunk 1 above)
Comment (joshuacords) [reply modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/facets/view/asset_entries.jsp]: I see you did it later

**Comment by joshuacords** [comment]
Hi @gustavolimav, besides the re-ordering, everything else looks good to me. We talked about the missing test for this facet but that can be handled under another subtask.

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test

**Comment by gustavolimav** [comment]
ci:test:sf

**Pull request closed without merging**